### PR TITLE
Updated documentation of synology integration to reflect the refactor and new switch

### PR DESCRIPTION
**Pull request by alexguzun**
**Description:**

Updated documentation of synology integration to reflect the refactor and new switch

**Pull request in home-assistant :** home-assistant/home-assistant#30970

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/synology.markdown [hunk 1]
```diff
@@ -1,38 +1,37 @@
...
 
 ```yaml
-# Minimum configuration.yaml entry
-camera:
-  - platform: synology
-    url: IP_ADDRESS_OF_SYNOLOGY_NAS
-    username: YOUR_USERNAME
-    password: YOUR_PASSWORD
+synology:
+  name: "Synology NAS"
```
Comment (frenck) [inline source/_integrations/synology.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Please remove `name` & `verify_ssl`

File: source/_integrations/synology.markdown [hunk 2]
```diff
@@ -53,30 +52,67 @@ timeout:
...
   required: false
   type: boolean
   default: true
 {% endconfiguration %}
 
+### Camera
+
+<div class='note'>
+
+Synology has disabled the livestreaming API and the integration is currently broken if you are using Surveillance Station version 8.2.3-5828.
```
Comment (frenck) [inline source/_integrations/synology.markdown]:
```suggestion
Synology has disabled the live streaming API and the integration is currently broken if you are using Surveillance Station version 8.2.3-5828.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @alexguzun! 

I've left 2 small comments, could you take a look? Thanks! 👍

File: source/_integrations/synology.markdown (hunk 2 above)
Comment (alexguzun) [reply source/_integrations/synology.markdown]: Changed. Thank you!

File: source/_integrations/synology.markdown (hunk 1 above)
Comment (alexguzun) [reply source/_integrations/synology.markdown]: Hi @frenck. Thank you for the review! I have removed the optional config parameters.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request closed without merging**